.test-infra/jenkins/dependency_check/dependency_check_report_generator_test.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+#
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Do we actually run these tests ? I couldn't find a reference to this. How about executing these tests as a part of the Jenkins job ?

Reply:
Why not running these a part of the Jenkins job ? I think if it's not very expensive we should run them (we do the same thing for performance tests). That will make us more confident on the results. If we don't run them somewhere automatically probably no one ever will and tests/code will become stale.

Reply:
Done. Added the unit test into the jenkins job. Fail the job if the unit test did not pass.